pkg/istio/mesh_config_test.go

```diff
@@ -199,3 +209,59 @@ func TestOSSMControlPlaneWrapper_SetMeshConfig(t *testing.T) {
 	assert.Equal(t, meshConfig.ExtensionProviders[0].Name, "custom-authorizer")
 	assert.Equal(t, meshConfig.ExtensionProviders[0].GetEnvoyExtAuthzGrpc().GetPort(), uint32(50051))
 }
+
+func TestSailWrapper_GetConfigObject(t *testing.T) {
+	ist := &istiov1alpha1.Istio{}
+	wrapper := NewSailWrapper(ist)
+
+	assert.Equal(t, wrapper.GetConfigObject(), ist)
+}
+
+func TestSailWrapper_GetMeshConfig(t *testing.T) {
+	structConfig := getStubbedMeshConfigStruct()
+	values := helm.HelmValues{}
+	if err := values.Set("meshConfig", structConfig.AsMap()); err != nil {
+		assert.NilError(t, err)
+	}
+	config := &istiov1alpha1.Istio{}
+	if err := config.Spec.SetValues(values); err != nil {
+		assert.NilError(t, err)
+	}
+	wrapper := NewSailWrapper(config)
+
+	meshConfig, err := wrapper.GetMeshConfig()
+	assert.NilError(t, err)
+	assert.Equal(t, meshConfig.ExtensionProviders[0].Name, "custom-authorizer")
+	assert.Equal(t, meshConfig.ExtensionProviders[0].GetEnvoyExtAuthzGrpc().GetPort(), uint32(50051))
+}
+
+func TestSailWrapper_GetMeshConfigInvalid(t *testing.T) {
+	structConfig := getInvalidStubbedMeshConfigStruct()
+	values := helm.HelmValues{}
+	if err := values.Set("meshConfig", structConfig.AsMap()); err != nil {
+		assert.NilError(t, err)
+	}
+	config := &istiov1alpha1.Istio{}
+	if err := config.Spec.SetValues(values); err != nil {
+		assert.NilError(t, err)
+	}
+	wrapper := NewSailWrapper(config)
+
+	meshConfig, err := wrapper.GetMeshConfig()
+	assert.Error(t, err, "proto: (line 1:2): unknown field \"invalid-field\"")
```

Comment:
I'll change this test.. https://github.com/protocolbuffers/protobuf-go/blob/master/internal/errors/errors.go#L27-L28